starlette/templating.py

```diff
@@ -117,16 +111,14 @@ def __init__(
 
     def _create_env(
         self,
-        directory: typing.Union[
-            str, PathLike, typing.Sequence[typing.Union[str, PathLike]]
-        ],
+        directory: "typing.Union[str, PathLike[typing.AnyStr], typing.Sequence[typing.Union[str, PathLike[typing.AnyStr]]]]",
```

Comment:
Is `PathLike[AnyStr]` fine? Or we could enforce str paths. I often see `PathLike[Any]` but doesn't feel right to me as `PathLike` is generic regarding `TypeVar("AnyStr_co", str, bytes, covariant=True)`